denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_kick_inv.dsc

```diff
@@ -0,0 +1,44 @@
+# -- KICK PANEL
+mod_kick_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>› <&d>Kick <&e><player.flag[amp_map].as_map.get[uuid].as_player.name>.
+  inventory: CHEST
+  size: 54
+  definitions:
+    x: <item[air]>
+    b1: <item[lime_stained_glass_pane].with[display_name=<&r>]>
+    b2: <item[green_stained_glass_pane].with[display_name=<&r>]>
+    back: <item[red_stained_glass_pane].with[display_name=<&c><&l>↩<&sp>Actions<&sp>panel;nbt=<list[to/actions]>]>
+    head: <item[mod_player_item]>
+  procedural items:
+    - define inventory:<list>
+    - foreach <list[1|2|3]> as:level:
+      - foreach <script[mod_kick_infractions].list_keys[<[level]>]> as:infraction:
+        - define item <item[mod_level<[level]>_item]>
+        - define name <[item].script.parsed_key[tag]><&sp><[infraction]>
+        - define lore <list[<&b>Level<&co><&sp><[item].script.parsed_key[colour]><[level]>]>
+        - define lore:|:<&e>Right<&sp>Click<&sp>to<&sp>kick<&co>
+        - define lore:|:<player.flag[amp_map].as_map.get[uuid].as_player.name>
+        - define lore:|:<&e>Clic<&sp>Droit<&sp>pour<&sp>un<&sp>coup<&co>
```

Comment:
These values being inserted into the list are single list values, `:->:` is preferred for single entry insertions over `:|:`